### PR TITLE
[Naitei] Write_Controller_Product_UnitTest

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```
Comment (nghinn1012) [inline app/views/devise/passwords/edit.html.erb]: i18n mấy cái title

File: app/views/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
```
Comment (nghinn1012) [inline app/views/devise/passwords/new.html.erb]: i18n luôn

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by nghinn1012 — APPROVED** [review]

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**